src/packet_analysis/Manager.h

```diff
@@ -40,7 +46,7 @@ class Manager : public plugin::ComponentManager<Tag, Component>
 	 * Second-stage initialization of the manager. This is called late
 	 * during Zeek's initialization after any scripts are processed.
```

Comment:
This should probably get an explanation of the argument in the documentation.

Reply:
Fixed, and also fixed it to not take a `std::optional` but just a string directly.